src/components/strategies/create/CreateStrategyTypeMenu.tsx

```diff
@@ -49,6 +49,16 @@ export const CreateStrategyTypeMenu: FC<UseStrategyCreateReturn> = ({
     selectedTabItems,
   } = useCreateStrategyTypeMenu(base?.address!, quote?.address!, strategyType);
 
+  useEffect(() => {
```

Comment:
not sure we need an effect for this. Why cant you set this as default value in the state?

Reply:
We have some case that we need the useEffect for:
1. user open the page, recurring as default and  first item selected -> click again on Recurring -> the first item won't be selected.
2. user open the page, recurring as default and  first item selected -> click on 2 Ranges -> click on Recurring again the first item won't be selected.

Update me if its not clear so we can have a short discord :)